packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1032,4 +1032,12 @@ export const threedtopia_xl = (model: ModelData): string[] => [
 model = threedtopia_xl.from_pretrained("${model.id}")
 model.generate(cond="path/to/image.png")`,
 ];
+
+export const hezar = (model: ModelData): string[] => [
+	`# !pip install hezar
```

Comment:
```suggestion
```

No need to add the pip install command. There will be a direct link to the Hezar repo so that users can get the full install explanations

Reply:
Done🤗